tests/pfcwd/conftest.py

```diff
@@ -27,6 +27,8 @@ def pytest_addoption(parser):
                      help='PFC WD storm restore interval')
     parser.addoption('--fake-storm', action='store', type=bool, default=True,
                      help='Fake storm for most ports instead of using pfc gen')
+    parser.addoption('--two-queues', action='store_true', default=False,
```

Comment:
Can you default this to True? We still want to keep the default behavior which enables us to test single vs multiple lossless priorities without having to increase the test run time by including separate testcases

Reply:
@neethajohn Main goal of this PR was to make execution of test more stable and predicted, using optional parameter `--two-queues` to specify if traffic should be sent to a single queue or to multiple queues. If I understand you correctly, you want to keep default behavior, when test will send traffic to single or multiple queues depending on some seed ? If that's the case then I will close this PR as it doesn't have any sense.

Reply:
@AndriiLozovyi , we can still have this PR with the changes I mentioned earlier. setting two_queues default to True and enhancing the check to add the second queue (check should have self.two_queues and seed). By setting two_queues to False during the test run, the test run will be predictable. For others, the default behavior remains unchanged

Reply:
@neethajohn Done, could you please take a look ?